src/types/events/base-events.ts

```diff
@@ -425,6 +426,42 @@ export interface FileUnsharedEvent {
   event_ts: string;
 }
 
+export interface FunctionParams {
```

Comment:
FYI: here is my investigation result: https://github.com/seratch/slack-edge/blob/0.10.1/src/request/payload/event.ts#L420-L427

type, name, is_required cannot be optional for sure.

There are a few more optional ones such as maximum, minimum, maxLength, and minLength. But you don't need to cover all the things this time.

Reply:
Thank you for verifying this! I've included this in the [pre-stable-fixes branch](https://github.com/slackapi/bolt-js/pull/2128/files#diff-7e67aa7d0c6331b94638639b911c10002ef75e4b5be11e75449ee92f3d2c7623R435). 🙇🏼‍♀️